src/Models/TaskVersion.php

```diff
@@ -95,4 +95,16 @@ public function getDeadlineAttribute($value): string
     {
         return Carbon::parse($value)->format('c');
     }
+
```

Comment:
These relationships won't work. TaskFile would need to belong to TaskVersion and to have a task version id. TaskFiles don't use soft delete either and the files are deleted from the AWS bucket. This means that we can't keep snapshots of what the files were at any point in time.